source/code/plugins/in_sudo_tail.rb

```diff
@@ -101,6 +101,10 @@ def receive_log(line)
     end
  
     def run_periodic
+      @paths = @path.split(' ').each {|path|
+        $log.info "Following tail of #{path}"
```

Comment:
Ideally, it will be only called when agent restarts, which is the same as using warn_once. So, it should be fine to use $log.info. But, yes, it's better to de-dup the paths if there are duplicates.